src/test/java/com/github/jhonnymertz/wkhtmltopdf/wrapper/unit/XvfbConfigTests.java

```diff
@@ -33,6 +33,6 @@ public void testXvfbParams() throws Exception {
         wc.setXvfbConfig(xvfbConfig);
         Pdf pdf = new Pdf(wc);
         pdf.addPageFromUrl("http://www.google.com");
-        Assert.assertThat("command params should contain xvfb params", pdf.getCommand(), containsString("xvfb-run --test-param --test-param2 test-value wkhtmltopdf http://www.google.com -"));
+        Assert.assertThat("command params should contain xvfb params", pdf.getCommand(), containsString("xvfb-run --test-param --test-param2 test-value wkhtmltopdf page http://www.google.com -"));
```

Comment:
Why does this `page` param get inserted without any changes in the test? What is the impact on the generated file?

Reply:
No impact on the generated file, if using wkhtmltopdf and passing it a file or url but without defining an object (toc,cover,page) type in front then page is implied as it is optional. The documentation contains: 
```
(page)? <input url/file name> [PAGE OPTION]...
```
Just for simplicity, I chose to always specify the object type in the generated command. Alternatively we could update the following:
```
// Page.java
    @Override
    public String SetObjectIdentifier()
    {
       // return "page";
       return "";
    }
```
This would mean we don't insert the 'page' object type into the command as it's implied if not provided.

Reply:
Got it, thanks for explaining! Yeah, a common troubleshooting activity when using this library is to copy the resulting command and try it directly on the bash console, people use it to make sure they achieved the expected command.

Considering the troubleshooting activity, I think it is better to keep it consistent with what the user wants to avoid leading to confusion while analyzing the generated command. Does it make sense?

Reply:
No worries, lets not specify 'page' as this is the default object type and let wkhtmltopdf imply this on it's own 👍 This also means that past command outputs will continue to match, meaning the above change can be reverted and we shouldn't impact any other user usages where they perhaps have tests in place to validate expected commands.